__tests__/api.test.js

```diff
@@ -31,3 +30,22 @@ describe('GET/api/topics', () => {
         });
     });
 });
+
+
+describe('GET/api', () => {
+    it('200: responds with all endpoints available', () => {
+        return request(app).get('/api')
+        .then((response) => {
+        expect(response.status).toBe(200);
+        })   
+    });
+
+    it('200: responds with the expected JSON describing all available endpoints', () => {
+        return request(app).get('/api')
+        .then((response) => {
```

Comment:
Instead of comparing the response to a hardcoded string, I recommend to check only if the object has the right properties. Or another and maybe easier way would be to require in the endpoints.json file here in this test file, then compare the response to that one with .toEqual().